plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -68,4 +68,7 @@ private MLCommonsSettings() {}
         .simpleString("plugins.ml_commons.exclude_nodes._name", Setting.Property.NodeScope, Setting.Property.Dynamic);
     public static final Setting<Boolean> ML_COMMONS_ALLOW_CUSTOM_DEPLOYMENT_PLAN = Setting
         .boolSetting("plugins.ml_commons.allow_custom_deployment_plan", false, Setting.Property.NodeScope, Setting.Property.Dynamic);
+
```

Comment:
Add comment about this settings.

Reply:
The Setting name is `VALIDATE_BACKEDN_ROLE` but the configuration is `filter_by_backend_roles` which are not consistent, can we rename one to match another?